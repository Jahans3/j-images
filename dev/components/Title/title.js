/**
 * Created by jahansj on 19/11/2016.
 */
import React, { Component } from 'react';
import s from './Title.css';

export default (props) => {
  return (
      <div>
        <h1 className={ s.titleText }>
          {
            props.text
          }
        </h1>
      </div>
  )
}