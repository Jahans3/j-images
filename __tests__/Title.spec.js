/**
 * Created by jahansj on 20/11/2016.
 */
import React from 'react';
import renderer from 'react-test-renderer';
import Title from '../dev/components/Title/Title';

test('Title', () => {
  const tree = renderer.create(
      <Title text="A Title" />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});