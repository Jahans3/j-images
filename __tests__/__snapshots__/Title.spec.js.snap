exports[`test Title 1`] = `
<div>
  <h1
    className={undefined}>
    A Title
  </h1>
</div>
`;
